pkg/continuoustest/write_read_series.go

```diff
@@ -84,8 +85,17 @@ func (t *WriteReadSeriesTest) Init(ctx context.Context, now time.Time) error {
 
 // Run implements Test.
 func (t *WriteReadSeriesTest) Run(ctx context.Context, now time.Time) {
+	// Configure the rate limiter to send a sample for each series per second. At startup, this test may catch up
+	// with previous missing writes: this rate limit reduce the chances to hit the ingestion limit on Mimir side.
```

Comment:
[Nit] Language improvement:
```suggestion
	// with previous missing writes: this rate limit reduces the chances to hit the ingestion limit on Mimir side.
```